### PR TITLE
Added an introductory paragraph for new users

### PR DESCRIPTION
**Pull request by ghost**
Most new users land on the quickstart after signing up for a playground account. I added some clarification around the value of our platform as well as what they will be building in the address book demo

### CODE REVIEW

**Comment by ghost** [comment]
@elof let me know what you think!

File: docs/quickstart.md [hunk 1]
```diff
@@ -3,9 +3,15 @@ sidebar_position: 10
...
+
+Great, time to jump right in! Let's get started with Macrometa by creating a global address book. In this tutorial, we will be creating a document [collection](https://macrometa.com/docs/collections/), inserting and querying some user data, and finally creating a fully operational API via [Query Workers](https://macrometa.com/docs/queryworkers/). This tutorial is Macrometa's version of "Hello, world!", is simplistic by design, and demonstrates how you can quickly use our stateful serverless back-end to run a globally distributed database with local read-write latencies around 50ms.
```
Comment (diana-macrometa) [inline docs/quickstart.md]:
```suggestion
Great, time to jump right in! Let's get started with Macrometa by creating a global address book. In this tutorial, you will be creating a document [collection](https://macrometa.com/docs/collections/), inserting and querying some user data, and finally creating a fully operational API via [Query Workers](https://macrometa.com/docs/queryworkers/). This tutorial is Macrometa's version of "Hello, world!", is simplistic by design, and demonstrates how you can quickly use our stateful serverless back-end to run a globally distributed database with local read-write latencies around 50ms. 
```

File: docs/quickstart.md [hunk 2]
```diff
@@ -3,9 +3,15 @@ sidebar_position: 10
...
+
+Great, time to jump right in! Let's get started with Macrometa by creating a global address book. In this tutorial, we will be creating a document [collection](https://macrometa.com/docs/collections/), inserting and querying some user data, and finally creating a fully operational API via [Query Workers](https://macrometa.com/docs/queryworkers/). This tutorial is Macrometa's version of "Hello, world!", is simplistic by design, and demonstrates how you can quickly use our stateful serverless back-end to run a globally distributed database with local read-write latencies around 50ms. 
+
+If you've already completed this tutorial, you can skip ahead to [add search to your application](https://macrometa.com/docs/search/getting-started)
```
Comment (diana-macrometa) [inline docs/quickstart.md]:
```suggestion
If you've already completed this tutorial, then you can [add search to your application](https://macrometa.com/docs/search/getting-started).
```

File: docs/quickstart.md [hunk 3]
```diff
@@ -148,4 +154,4 @@ IN addresses
 
 You have a full functional API for your application. [We made a front-end for you to take your new back-end for a spin](https://github.com/Macrometacorp/tutorial-addressbook-streams).
```
Comment (diana-macrometa) [inline docs/quickstart.md]:
```suggestion
You have a fully-functional API for your application. [We made a front-end for you to take your new back-end for a spin](https://github.com/Macrometacorp/tutorial-addressbook-streams).
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
Remember that in technical writing, we generally speak to "you" rather than "we."

**Comment by ghost** [comment]
@diana-macrometa any update on this? Are we just waiting for other reviewers?

**Review by diana-macrometa — APPROVED** [review]

**Comment by diana-macrometa** [comment]
LGTM, I must have lost track of it.

**Comment by ghost** [comment]
@diana-macrometa I'm not quite sure what to do with the conflict that's listed above, but it's minor and silly

**Pull request merged**